packages/chip-input/src/ux-chip-input.ts

```diff
@@ -1,18 +1,28 @@
 import { customElement, bindable } from 'aurelia-templating';
-import { DOM } from 'aurelia-pal';
-import { bindingMode } from 'aurelia-binding';
+import { bindingMode, observable, computedFrom } from 'aurelia-binding';
 import { inject } from 'aurelia-dependency-injection';
-import { StyleEngine, UxComponent, normalizeBooleanAttribute } from '@aurelia-ux/core';
+import { StyleEngine, UxInputComponent, normalizeBooleanAttribute } from '@aurelia-ux/core';
 import { UxChipInputTheme } from './ux-chip-input-theme';
+// tslint:disable-next-line: no-submodule-imports
+import '@aurelia-ux/core/styles/ux-input-component.css';
+// tslint:disable-next-line: no-submodule-imports
+import '@aurelia-ux/core/styles/ux-input-component--outline.css';
 
 @inject(Element, StyleEngine)
 @customElement('ux-chip-input')
-export class UxChipInput implements UxComponent {
+export class UxChipInput implements UxInputComponent {
   @bindable public disabled: boolean | string = false;
   @bindable public readonly: boolean | string = false;
   @bindable public theme: UxChipInputTheme;
-  @bindable public label: any;
+  @bindable public label: string;
+  @bindable public placeholder: string;
   @bindable public separator: string = ', ';
+  @bindable public variant: 'filled' | 'outline' = 'filled';
+  @bindable public chipVariant: 'filled' | 'outline' = 'filled';
+  @bindable public dense: any = false;
```

Comment:
`dense: boolean` ?

Reply:
Hum... I wanted that we could also use this as an attribute without value, hence the normalizeBooleanAttribute in the bind() but I’m not sure I did this right. Probably need to fix this !

Reply:
I see, probably no need to fix then. Maybe a comment will do the explanation job easily

Reply:
Yes, but might need to check/fix to ensure that what I try to achieve actually works :-/